mesonbuild/backend/xcodebackend.py

```diff
@@ -735,6 +735,7 @@ def generate_xc_build_configuration(self):
                     product_name = target.get_basename()
                 ldargs += target.link_args
                 cargs = []
+                cargs.append('-I.')
```

Comment:
this doesn't seem right, isn't this ignoring the implicit_include_paths argument?

Reply:
Hi Dylan,
When looking at the build output in xcode, it first does cd to the source directory,
then it runs clang to compile the code.
This basically adds the source directory as an include path.
Otherwise I get a compile error.
I’m not sure if this is the best way though.
Is there an API for getting the source directory?

Thanks,
Jeff